src/formatDate.js

```diff
@@ -49,8 +49,59 @@
  * @returns {string}
  */
 
+function yearFormating(year) {
+  return year >= 30 ? `19${year}` : `20${year}`;
+}
+
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+
+  let day;
+  let month;
+  let year;
+  const separator = toFormat[toFormat.length - 1];
```

Comment:
sorry for missunderstanding, I meant that the separator shouldnt be in the object, but it was great that day, month and year were in that object